packages/core/core/src/Strapi.ts

```diff
@@ -243,22 +243,24 @@ class Strapi extends Container implements Core.Strapi {
 
   // TODO: split into more providers
   registerInternalServices() {
+    const config = createConfigProvider(this.internal_config, this);
+
+    const logger = createLogger({
+      level: 'http', // Strapi defaults to level 'http'
+      ...config.get('logger'), // DEPRECATED
```

Comment:
Do we want to soft deprecate it and remove it in v6, or do we want to create a codemod for that and remove this line?

Reply:
(Wondering if it's even possible to create a codemod for it :thinking: )

Reply:
I just moved the code I didn't think about it at all :D